### PR TITLE
*: add an unsupported optimistic shard DDL statement

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

add an unsupported optimistic shard DDL statement

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@lance6716 @CharLotteiu PTAL

**Comment by lance6716** [comment]
LGTM

**Comment by CharLotteiu** [comment]
LGTM. @yikeke PTAL.

File: en/feature-shard-merge-optimistic.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Currently, the following statements are not supported in the optimistic mode:
 
 - `ALTER TABLE table_name ADD COLUMN column_name datatype NOT NULL` (To add a `NOT NULL` column without a default value).
 - `ALTER TABLE table_name ADD COLUMN column_name datetime DEFAULT NOW()` (To add a column with a varying value).
+- `ALTER TABLE table_name ADD COLUMN col1 INT, DROP COLUMN col2` (Contain both `ADD COLUMN` and `DROP COLUMN` in one DDL statement).
```
Comment (yikeke) [inline en/feature-shard-merge-optimistic.md:42]:
```suggestion
- `ALTER TABLE table_name ADD COLUMN col1 INT, DROP COLUMN col2` (Contains both `ADD COLUMN` and `DROP COLUMN` in one DDL statement).
```

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**